changelog/changelog.go

```diff
@@ -1,12 +1,22 @@
 package changelog
 
 import (
+	"fmt"
 	"text/template"
 )
 
 const (
-	tplStr = `### {{.OldRelease}} -> {{.NewRelease}}
+	tplStr = `{{ if and .OldRelease .NewRelease }}
+### {{.OldRelease}} -> {{.NewRelease}}
+{{end}}
+
+{{ if (len .Migrations) gt 0 -}}
+### Migrations
```

Comment:
Is `Migrations` the right term for this? Maybe `Updated Components` or `Releases`? I just thought it read a bit strangely:

### Migrations

- builder v2.5.4 -> v2.5.5
- controller v2.8.0 -> v2.8.1
- postgres v2.2.5 -> v2.2.6
- registry v2.2.3 -> v2.2.4

#### Fixes

- [`4745179`](https://github.com/deis/builder/commit/4745179d9487ced5607cd7398a55c5488fabf5c9) (builder) - charts: Use the common storage secret